bokehjs/src/lib/core/enums.ts

```diff
@@ -37,6 +37,9 @@ export const Direction = Enum("clock", "anticlock")
 export type Distribution = "uniform" | "normal"
 export const Distribution = Enum("uniform", "normal")
 
+export type Extend = "neither" | "min" | "max" | "both"
+export const Extend = Enum("neither", "min", "max", "both")
```

Comment:
`Extend` is a too generic name, i.e. extend what? The same goes for the property name.

Reply:
`"none"` would be more idiomatic to bokeh than `"neither"`.